### PR TITLE
Updating Fonos voice application examples

### PR DESCRIPTION
**Pull request by speedymonster**

### CODE REVIEW

**Comment by speedymonster** [comment]
I have read the CLA Document and I hereby sign the CLA

File: examples/presidential-poll/index.js [hunk 1]
```diff
@@ -12,19 +12,20 @@ const voiceServerConfig = {
   base: ""
 };
 const candidates = {
-  1: {name: "Donald Trump"},
-  2: {name: "Hillary Clinton"},
-  3: {name: "Beny Sanders"}
+  1: { name: "Donald Trump" },
```
Comment (wardner) [inline examples/presidential-poll/index.js]: Delete the spaces between the brackets.

**Review by wardner — CHANGES_REQUESTED** [review]
Waiting for changes to be done

File: examples/hello-monkeys/index.js [hunk 2]
```diff
@@ -1,4 +1,7 @@
 /**
  * The simplest Voice Application you can build in Fonos
  */
-module.exports = (chan) => chan.play("tt-monkeys");
+const { VoiceServer } = require("@fonos/voice");
```
Comment (wardner) [inline examples/hello-monkeys/index.js]: Remove the space between brackets.

**Review by wardner — CHANGES_REQUESTED** [review]
Waiting for changes

**Review by wardner — COMMENTED** [review]
Looks good to me @psanders

**Pull request merged**